pkg/feeds/npm/npm.go

```diff
@@ -153,15 +180,22 @@ func fetchPackage(httpClient *http.Client, baseURL, pkgTitle string) ([]*Package
 		return versionSlice[j].CreatedDate.Before(versionSlice[i].CreatedDate)
 	})
 
+	if etag != "" {
+		feed.cache.Add(versionURL, &cacheEntry{
```

Comment:
nit: add comment explaining the implication of etag being blank - does it mean it's not in the cache and had to be retrieved?

Reply:
Done